### DISCUSSION TITLE
save to html

### DISCUSSION THREAD

**Discussion by hbowkett** [opengeos/leafmap#23] [Q&A] [answered]
is there an example of saving to html. I get Exception: [Errno 2] No such file or directory: '' and would like to test a notebook that works

**Comment by giswqs** [comment, marked as answer]
This bug has been fixed. Please update the package using `leafmap.update_package()`. 

**For folium**
```python
import leafmap.foliumap as leafmap
m = leafmap.Map()
m.save("mymap.html")
```
**For ipyleaflet** (Notebook: it seems ipyleaflet has a bug right now. You might get a blank map. See https://github.com/jupyter-widgets/ipyleaflet/issues/841)
```python
import leafmap
m = leafmap.Map()
# m.save("mymap.html") #this one should also work
m.to_html("mymap.html")
```

**Comment by hbowkett** [comment]
Many thanks for this.
Yes it is sorted now for me.
I switched to your package because a bug in ipyleaflet nudged me into
looking at other options.
For me leafmap is just what I need - so many thanks for developing it
regards
Helen

regards
Helen
